.github/workflows/build-m1-wheel.yml

```diff
@@ -28,7 +39,7 @@ jobs:
 
     - name: Build m1 wheels
       run: |
-        arch -arm64 python -m venv venv
+        arch -arm64 python3 -m venv venv
```

Comment:
the first problem was that this invoked `python2` which didn't have the `venv` module